apps/settings/js/bluetooth.js

```diff
@@ -664,13 +666,14 @@ navigator.mozL10n.once(function bluetoothSettings() {
         }
       } else {
         // show pair process fail.
-        var msg = _('error-pair-title');
+        var l10nId = 'error-pair-title';
         if (errorMessage === 'Repeated Attempts') {
-          msg = msg + '\n' + _('error-pair-toofast');
+          l10nId = 'error-pair-toofast';
         } else if (errorMessage === 'Authentication Failed') {
-          msg = msg + '\n' + _('error-pair-pincode');
+          l10nId = 'error-pair-pincode';
```

Comment:
Yep, I noticed there are repeated words but it was UX design of "title" and "body", so we can't remove one of them, otherwise the UI will be a bit weird.